### PR TITLE
C353209 Export failed when using .csv file with non-existent UUIDs (Spitfire)

### PR DESCRIPTION
**Pull request by arsenhovh**
[Test Rail](https://foliotest.testrail.io/index.php?/cases/view/353209)
[Allure Report](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5277)

image

### CODE REVIEW

File: cypress/support/fragments/marcAuthority/marcAuthority.js [hunk 1]
```diff
@@ -87,6 +88,13 @@ export default {
     cy.do(Button('Edit').click());
     QuickMarcEditorWindow.waitLoading();
   },
+  delete: () => {
+    cy.do([
+      rootSection.find(Button('Actions')).click(),
+      Button('Delete').click(),
+      Modal('Confirm deletion of authority record').find(Button('Delete')).click(),
+    ]);
+  },
```
Comment (Sherzod-Kenjaev) [inline cypress/support/fragments/marcAuthority/marcAuthority.js:97]: There are already existing methods to delete marc authority record in marcAuthoritiesDelete.js file please use them.

File: cypress/support/fragments/marcAuthority/marcAuthorities.js [hunk 2]
```diff
@@ -388,6 +388,11 @@ export default {
     cy.do(Checkbox(value).click());
   },
 
+  downloadSelectedRecordWithRowIdx(checkBoxNumber = 1) {
+    cy.get(`div[class^="mclRow--"]:nth-child(${checkBoxNumber}) input[type="checkbox"]`).click();
+    cy.do([actionsButton.click(), Button('Export selected records (CSV/MARC)').click()]);
```
Comment (Sherzod-Kenjaev) [inline cypress/support/fragments/marcAuthority/marcAuthorities.js:393]: Move interactor  Button('Export selected records (CSV/MARC)') to const variable please.

File: cypress/e2e/data-export/export-failed-using-non-existent-UUIDs.cy.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  before(() => {
+    cy.createTempUser([
+      Permissions.settingsDataImportEnabled.gui,
+      Permissions.moduleDataImportEnabled.gui,
+      Permissions.uiMarcAuthoritiesAuthorityRecordView.gui,
+      Permissions.uiMarcAuthoritiesAuthorityRecordEdit.gui,
+      Permissions.uiMarcAuthoritiesAuthorityRecordDelete.gui,
+      Permissions.uiQuickMarcQuickMarcAuthoritiesEditorAll.gui,
+      Permissions.dataExportAll.gui,
+      Permissions.dataExportEnableModule.gui,
```
Comment (Sherzod-Kenjaev) [inline cypress/e2e/data-export/export-failed-using-non-existent-UUIDs.cy.js:33]: Use required permissions, in test case it is asked to use three permissions.

**Review by Sherzod-Kenjaev — CHANGES_REQUESTED** [review]

**Review by Sherzod-Kenjaev — APPROVED** [review]

**Review by zentestuken — COMMENTED** [review]
Please fix linter issues

**Review by zentestuken — APPROVED** [review]

**Pull request merged**